stumpy/aamp.py

```diff
@@ -438,4 +438,4 @@ def aamp(T_A, m, T_B=None, ignore_trivial=True, p=2.0, k=1):
 
     core._check_P(out[:, 0])
 
-    return mparray(out, m=m, k=k)
+    return mparray(out, m, k, config.STUMPY_EXCL_ZONE_DENOM)
```

Comment:
Passing arguments without their name is cleaner in this case! I totally missed it! Nice!!

Reply:
Yeah, since I made every argument a "required" argument then all names can be dropped.